test/e2e/run.sh

```diff
@@ -99,9 +99,6 @@ else
   exit
 fi
 
-# Remove after https://github.com/kubernetes/ingress-nginx/pull/4271 is merged
```

Comment:
Only this comment can be removed. The multi-arch image is built only for releases, not e2e tests

Reply:
oops I misunderstood, fixing now